### PR TITLE
Created new Spring project and setup Github Action

### PR DESCRIPTION
**Pull request by singaltanmay**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+		<relativePath/> <!-- lookup parent from repository -->
+	</parent>
+	<groupId>uk.ac.ebi.eva</groupId>
+	<artifactId>contig-alias</artifactId>
+	<version>0.0.1-SNAPSHOT</version>
+	<name>contig-alias</name>
+	<description>Service to provide synonyms of chromosome/contig identifiers </description>
+
+	<properties>
+		<java.version>11</java.version>
```
Comment (jmmut) [inline pom.xml:18]: Just to let you know, most of our projects are still on java 8, but I think it's a good idea to start trying projects on java 11.

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+		</dependency>
+
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-test</artifactId>
+			<scope>test</scope>
+			<exclusions>
+				<exclusion>
+					<groupId>org.junit.vintage</groupId>
+					<artifactId>junit-vintage-engine</artifactId>
```
Comment (jmmut) [inline pom.xml:34]: Did the start.spring.io project include this by default? Anyway, I can imagine the reason to do this.

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,31 @@
+HELP.md
+target/
+!.mvn/wrapper/maven-wrapper.jar
```
Comment (jmmut) [inline .gitignore:3]: there is some stuff in this PR about maven wrapper (which we don't use in other projects), but I don't mind keeping it here if you are going to use it. The only benefit of keeping it is that it doesn't force the user to install maven.

File: pom.xml (hunk 2 above)
Comment (singaltanmay) [reply pom.xml:34]: Yes start.spring.io included this by deafult

File: .gitignore (hunk 3 above)
Comment (singaltanmay) [reply .gitignore:3]: This .gitignore file was also generated by start.spring.io but wouldn't the user need maven to build this project anyway to resolve dependencies in pom.xml?

Comment (jmmut) [reply .gitignore:3]: the thing is, users have 2 options:
1. install maven (through apt-get for instance). then the maven wrapper stuff is not used.
2. use mvnw. this doesn't require the user to install maven. It checks if it's installed, and if not, mvnw downloads maven and uses it.

**Pull request merged**